project/ws/app/src/lib/routes/home/routes/approvals/approval-pending/approval-pending.component.ts

```diff
@@ -161,6 +163,8 @@ export class ApprovalPendingComponent implements OnInit, OnDestroy {
 
       this.apprService.getApprovals(req).subscribe(res => {
         // this.data = []
+        this.totalprofileVerificationRecords = res.result.count
```

Comment:
please put a check for res, if res and res.result like that

Reply:
putted the condition